docs/commands/stats.md

```diff
@@ -1,122 +1,142 @@
-    Operations,
-    Tags
-```
+## Introduction
 
+The `stats` command provides statistics about the structure of one or more API definition files. Statistics are calculated using the counting logic from the `StatsVisitor` module. The `stats` command can generate statistics for the following metrics:
 
-### `stats` usage
+* `References`
+* `External Documents`
+* `Schemas`
+* `Parameters`
+* `Links`
+* `Path Items`
+* `Operations`
+* `Tags`
 
+## Usage
 
-```shell
-Positionals:
-  entrypoint                                                 [string] [required]
-Options:
-  --version            Show version number.                            [boolean]
-  --help               Show help.                                      [boolean]
-  --format             Use a specific output format.
-                               [choices: "stylish", "json"] [default: "stylish"]
-  --config             Specify path to the config file.                 [string]
+```bash
+openapi stats <entrypoint>
+openapi stats <entrypoint> [--format] [--config=<path>]
+openapi stats --version
 ```
 
+## Options
 
-The command:
+Option                    | Type      | Required     | Default     | Description
+--------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`              | `string`  | yes          | -           | Path to the API definition filename that you want to calculate statistics for. Instead of full paths, you can use aliases assigned in the `apiDefinitions` section within your `.redocly.yaml` configuration file as entrypoints. Refer to [the entrypoint section below](#entrypoint) for more options.
+`--config`                | `string`  | no           | -           | Specify path to the [config file](#custom-configuration-file)
+`--format`                | `string`  | no           | `stylish`   | Format for the output.<br />**Possible values:** `stylish`, `json`
+`--help`                  | `boolean` | no           | -           | Show help
+`--version`               | `boolean` | no           | -           | Show version number
 
+## Examples
 
-```bash
-openapi stats openapi/petstore.yaml
-```
-
+### Entrypoint
 
-The output will be:
+The `stats` command behaves differently depending on how you pass entrypoint to it and whether the [configuration file](#custom-configuration-file) exists. There are the following options:
 
+#### Pass entrypoint directly
 
-```shell
-🚗 References: 12
-📦 External Documents: 3
-📈 Schemas: 10
-👉 Parameters: 9
-🔗 Links: 0
-➡️ Path Items: 16
-👷 Operations: 22
-🔖 Tags: 5
+```bash
+openapi stats openapi/openapi.yaml
 ```
 
+In this case, `stats` will show statistics for the definition that was passed to the command. The configuration file is ignored.
 
-### Options
+#### Pass entrypoints via configuration file
 
-#### Format
+Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints. For example, `petstore`:
```

Comment:
```suggestion
Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints. 
```